Tests/Docker/Container/ProcessTest.php

```diff
@@ -121,34 +94,10 @@ public function testFilterLargeOutput(): void
 fwrite(STDERR, 'b' . str_repeat(substr('😀', 0, 1), 90*(10**6)) . PHP_EOL);
 PHP
         );
+        $outputFilter = new OutputFilter(10000);
         $process = new Process(['php', $this->temp->getTmpFolder() . '/run.php']);
+        $process->setOutputFilter($outputFilter);
         $process->run();
-        var_dump(memory_get_peak_usage(true));
-        self::assertSame('a [trimmed]', $process->getOutput());
-        self::assertSame('b [trimmed]', $process->getErrorOutput());
-    }
-
-    public function testFilterLargeOutputCallback(): void
-    {
-        file_put_contents($this->temp->getTmpFolder() . '/run.php', <<<'PHP'
-<?php
-
-fwrite(STDOUT, 'a' . str_repeat(substr('😀', 0, 1), 90*(10**6)) . PHP_EOL);
-fwrite(STDERR, 'b' . str_repeat(substr('😀', 0, 1), 90*(10**6)) . PHP_EOL);
-PHP
-        );
-        $process = new Process(['php', $this->temp->getTmpFolder() . '/run.php']);
-        $events = [];
-        $process->run(function ($type, $e) use (&$events) {
-            return $events[$type][] = $e;
-        });
-        var_dump(memory_get_peak_usage(true));
-        // there can be any number of `[trimmed]` events depending on how the garbage is broken into buffers,
-        // which depends on system mood
-        self::assertGreaterThanOrEqual(1, count($events['out']));
-        self::assertGreaterThanOrEqual(1, count($events['err']));
-        self::assertStringStartsWith('a', $events['out'][0]);
-        self::assertStringStartsWith('b', $events['err'][0]);
```

Comment:
IC, this tests the same thing, but does it?  since there is no space in the emojis it will trim them all, so maybe 
```
fwrite(STDERR, 'b😀c' . str_repeat(substr('😀', 0, 1), 90*(10**6)) . PHP_EOL);
```
could be more useful?  that way it'll test both the broken emoji ignore and the trim?

Reply:
I don't understand :)